docs/faq.md

```diff
@@ -84,9 +74,17 @@
 
   The `--sync-period` option specifies the minimum frequency at which watched resources are reconciled. The synchronization period should be aligned with the number of managed Customer Resources. If the period is too low and the number of managed resources is too high, you may observe slowness in synchronization.
 
-- **Does the Operator work with Terraform Enterprise?**
+- **Does the Operator work with Terraform Enterprise / TFE?**
+
+  Yes, the operator can be configured for the custom TFE API endpoint using the `operator.tfeAddress` value in the Helm chart. This value should be a valid URL including the protocol(`https://`), for the API of a Terraform Enterprise instance. Once the `operator.tfeAddress` attribute is set, the operator will no longer access the public Terraform Cloud, but rather the private Terraform Enterprise instance.
```

Comment:
```suggestion
  Yes, the operator can be configured to use the custom TFE API endpoint using the [`operator.tfeAddress`](https://github.com/hashicorp/terraform-cloud-operator/blob/main/charts/terraform-cloud-operator/values.yaml#L28-L29) value in the Helm chart. This value should be a valid URL including the protocol(`https://`), for the API of a Terraform Enterprise instance. Once the `operator.tfeAddress` attribute is set, the operator will no longer access the public Terraform Cloud, but rather the private Terraform Enterprise instance.
```

Reply:
Same as above. Instead of point to specific line in the `value.yaml` file, shall we point to the [Values](https://github.com/hashicorp/terraform-cloud-operator/tree/main/charts/terraform-cloud-operator#values) in the Helm Chart README file?